crates/notedeck_columns/src/ui/note/post.rs

```diff
@@ -318,6 +324,28 @@ impl<'a, 'd> PostView<'a, 'd> {
     }
 
     pub fn ui(&mut self, txn: &Transaction, ui: &mut egui::Ui) -> PostResponse {
+        #[cfg(any(target_os = "android"))]
+        {
+            while let Some((display_name, content)) = get_next_selected_file() {
+                use std::path::PathBuf;
+
+                match MediaPath::new(PathBuf::from(display_name)) {
+                    Ok(media_path) => {
+                        let promise = nostrbuild_nip96_upload_bytes(
+                            self.poster.secret_key.secret_bytes(),
+                            media_path,
+                            content,
+                        );
+                        self.draft.uploading_media.push(promise);
+                    }
+                    Err(e) => {
+                        error!("{e}");
+                        self.draft.upload_errors.push(e.to_string());
+                    }
+                }
+            }
+        }
```

Comment:
platform details should not be exposed in app code. we shouldn't have different code paths unless there's a good reason to?